bootstrap-5/package.json

```diff
@@ -1,19 +1,17 @@
 {
-  "name": "node-starter",
+  "name": "bootstrap-5-demo",
   "version": "0.0.0",
+  "private": true,
   "scripts": {
-    "start": "webpack serve"
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
   },
```

Comment:
Converted the Bootstrap 5 example from webpack to Vite.
Simpler less verbose config, and HMR for free.